src/plugins/vm/VirtualMachineImpl.cpp

```diff
@@ -111,7 +133,7 @@ double VMModel::next_occuring_event(double now)
 
 VirtualMachineImpl::VirtualMachineImpl(simgrid::s4u::VirtualMachine* piface, simgrid::s4u::Host* host_PM,
                                        int core_amount, size_t ramsize)
-    : HostImpl(piface), physical_host_(host_PM), core_amount_(core_amount), ramsize_(ramsize)
+    : HostImpl(piface), physical_host_(host_PM), core_amount_(core_amount), ramsize_(ramsize), user_bound_(std::numeric_limits<double>::max())
```

Comment:
```suggestion
    : HostImpl(piface), physical_host_(host_PM), core_amount_(core_amount), user_bound_(std::numeric_limits<double>::max()), ramsize_(ramsize)
```

Reply:
This should fix the compilation error that you get when activating the warnings:
https://travis-ci.org/simgrid/simgrid/jobs/457571304#L1665